src/main/scala/org/broadinstitute/dsde/agora/server/webservice/PerRequest.scala

```diff
@@ -53,6 +54,10 @@ trait PerRequest extends Actor {
 
   override val supervisorStrategy =
     OneForOneStrategy() {
+      case authException: AgoraAuthorizationException =>
+        system.log.error(authException, "Authorization exception processing request:" + r.request.uri)
```

Comment:
error?  Or info?  It's not really a system error that a user tried to foo a bar without access. It's just dumbfoolery on their part